packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -288,5 +295,11 @@ const getStyles = (theme: GrafanaTheme2) => {
     rightAligned: css({
       marginLeft: 'auto',
     }),
+    titleItems: css({
+      display: 'flex',
+      alignItems: 'center',
+      overflow: 'hidden',
+      padding: theme.spacing(1),
```

Comment:
this padding I think should be removed, each item has what ever padding it needs?

It looks like this padding extends outside the bounds of the header:
<img width="402" alt="Screenshot 2023-01-10 at 17 41 05" src="https://user-images.githubusercontent.com/10999/211623588-e280937a-b2a6-43f1-b566-ee3a18515e97.png">

Reply:
Yes, I added this extra padding to prevent the focus ring from disappearing when the panel header gets smaller and the `overflow: hidden` from the title items kicks in, I didn't find a better solution to solve this problem, I saw this issue from [w3c](https://github.com/w3c/csswg-drafts/issues/3417).

https://user-images.githubusercontent.com/239999/211778870-27286ed6-0ac4-41fb-86f2-2a6cd4a565e5.mp4

Reply:
ah, interesting. well if the padding does not cause issues then I guess it's fine, just looked strange when it extended the bounds :) 

There lack of spacing / padding (or gap) between title and the first title item (or description)